mypy/errorcodes.py

```diff
@@ -262,6 +262,12 @@ def __hash__(self) -> int:
     default_enabled=False,
 )
 
+OVERLOADED_FUNCTION_MATCHING: Final[ErrorCode] = ErrorCode(
+    "overloaded-function-matching",
+    "Warn user about signature matching for overloaded functions.",
+    "General",
```

Comment:
if you do this, it makes it backwards compatible: people who want more specific `type: ignore` comments can use the new error code, but other people's existing `type: ignore` comments won't be broken

```suggestion
    "General",
    sub_code_of=MISC,
```

Reply:
great, thanks for pointing that out!